apps/core/components/PlacePicker/PlacePickerContent.js

```diff
@@ -158,12 +158,14 @@ export default createRefetchContainer(
           input: { type: "LocationsByTermInput!", defaultValue: { term: "" } }
         ) {
         locationsByTerm(input: $input) {
-          edges {
-            node {
-              id
-              name
-              locationId
-              type
+          ... on LocationConnection {
```

Comment:
So currently, you are not using the errors you just provided? Will that be part of another PR?

Reply:
yes this is just about the GraphQL part right now